includes/manifest.php

```diff
@@ -0,0 +1,194 @@
+<?php
+/**
+ * HM Platform Plugin Manifest.
+ */
+
+namespace HM\Platform;
+
+// Cavalcade.
+Plugin::register( 'cavalcade', 'plugins/cavalcade/plugin.php' )
+      ->load_on( true )
+      ->load_with( function ( $plugin ) {
+	      // Load the Cavalcade Runner CloudWatch extension.
+	      // This is loaded on the Cavalcade-Runner, not WordPress, crazy I know.
+	      if ( class_exists( 'HM\\Cavalcade\\Runner\\Runner' ) && HM_ENV_TYPE !== 'local' ) {
+		      require_once ROOT_DIR . '/lib/aws-sdk/aws-autoloader.php';
+		      require_once ROOT_DIR . '/lib/cavalcade-runner-to-cloudwatch/plugin.php';
+	      }
+
+	      // Load plugin on normal hook.
+	      add_action( 'muplugins_loaded', function () use ( $plugin ) {
+
+		      // Force DISABLE_WP_CRON for Cavalcade.
+		      if ( ! defined( 'DISABLE_WP_CRON' ) ) {
+			      define( 'DISABLE_WP_CRON', true );
+		      }
+
+		      require $plugin->get_file();
+	      } );
+      } )
+      ->enabled( true )
+      ->set_data( function () {
+	      return [
+		      'title'       => __( 'Cavalcade', 'hm-platform' ),
+		      'description' => __( 'Scalable background tasks for multi-server hosting setups.', 'hm-platform' ),
+		      'repository'  => 'humanmade/cavalcade',
```

Comment:
The format here looks different than the `repository` format in package.json?

Reply:
doh!